packages/icons/icons.json

```
@@ -772,84 +772,84 @@
     {
         "displayName": "Disable",
         "iconName": "disable",
-        "tags": "off, circle, remove",
+        "tags": "off, circle, remove, not, do not enter, no, anti, cancel, exit, no selection, do not",
         "group": "action",
         "codepoint": 61821
     },
     {
         "displayName": "Unpin",
         "iconName": "unpin",
-        "tags": "map, position, safety pin, detach",
+        "tags": "map, position, safety pin, detach, branch, git, version, pushpin, pin, thumbtack",
         "group": "action",
         "codepoint": 62214
     },
     {
         "displayName": "Flows",
         "iconName": "flows",
-        "tags": "arrows, direction, links",
+        "tags": "arrows, direction, links, proceed, link, leads to, flow, connection, processing, moving to, connect, pipeline",
         "group": "data",
         "codepoint": 61879
     },
     {
         "displayName": "New text box",
         "iconName": "new-text-box",
-        "tags": "text box, edit, new, create",
+        "tags": "text box, edit, new, create, new font",
         "group": "action",
         "codepoint": 62041
     },
     {
         "displayName": "New link",
         "iconName": "new-link",
-        "tags": "create, add, plus, links",
+        "tags": "create, add, plus, links, add link, add to chain, insert, addevent",
         "group": "action",
         "codepoint": 62037
     },
     {
         "displayName": "New object",
         "iconName": "new-object",
-        "tags": "create, add, plus, objects, circle",
+        "tags": "create, add, plus, objects, circle, add object, add element, new entity",
         "group": "action",
         "codepoint": 62038
     },
     {
         "displayName": "Path search",
         "iconName": "path-search",
-        "tags": "map, magnifying glass, position, location",
+        "tags": "map, magnifying glass, position, location, search, map search, locate",
```

some of these are redundant

```suggestion
        "tags": "map, magnifying glass, position, location, locate",
```